Level-2/trmv/trmv.cpp

```diff
@@ -53,7 +89,7 @@ int main(int argc, char** argv)
     size_t sizeX, absIncx;
 
     rocblas_int lda   = N;
-    int      sizeA = lda * int(N);
+    int         sizeA = int64_t(lda) * N;
```

Comment:
sizeA should then be int64_t , but you kept sizeX unsigned so unsigned would be fine too.